readthedocs_ext/external_version_warning.py

```diff
@@ -16,6 +17,16 @@ def process_external_version_warning_banner(app, doctree, fromdocname):
     If the version type is external this will show a warning banner
     at the top of each page of the documentation.
     """
+    # For unknown reason Sphinx emits this not only for
+    # actual documents, but also, e.g., docutils.nodes.bullet_list
+    # We don't want this warning to appear in random places,
+    # thus we return here if the doctree doesnt represent a document
+    # Details:
+    # - https://github.com/readthedocs/readthedocs-sphinx-ext/issues/113
+    # - https://github.com/readthedocs/readthedocs-sphinx-ext/pull/114
+    if not isinstance(doctree, addnodes.document):
```

Comment:
I think you can use docutils here instead of sphinx's addnodes.

```suggestion
    if not isinstance(doctree, docutils.document):
```

https://github.com/sphinx-doc/sphinx/blob/2f1d92773cc6e2857c6d4a0fd30caecbbfd5d4b3/sphinx/addnodes.py#L20-L20